src/Frontend-GNOME/Views/Chats/ChatView.cs

```diff
@@ -529,7 +529,7 @@ public virtual void Sync()
             DateTime start, stop;
             start = DateTime.UtcNow;
             // REMOTING CALL
-            SyncedMessages = _ChatModel.Messages;
+            SyncedMessages = _ChatModel.MessageBuffer;
```

Comment:
MessageBuffer does not implement [Serializable] in older smuxi-server versions, so that API shouldn't be used even though the Messages property says it is deprecated.